lib/period.js

```diff
@@ -30,7 +41,11 @@ function Period (str) {
 
   const [, year, term, period] = matching
 
-  return Period.fromObject({ year, term, period })
+  return Period.fromObject({
```

Comment:
I agree with Rasmus, I don't really understand what's going on here. What's this `Period.fromObject` function? I tried to search the web for a description about fromObject functions, but couldn't find anything. Can you point me to where I can find information about it @exacs ?